packages/website/src/theme/NotFound/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { useLocation } from '@docusaurus/router';
+import { PageMetadata } from '@docusaurus/theme-common';
+import Layout from '@theme/Layout';
+import React from 'react';
+
+import styles from './styles.module.css';
+
+export default function NotFound(): React.JSX.Element {
+  const location = useLocation();
+
+  return (
+    <>
+      <PageMetadata title="Page Not Found" />
+      <Layout>
+        <main className="container margin-vert--xl">
+          <div className="row">
+            <div className="col col--8 col--offset-2">
+              <h1 className={styles.title}>
+                <strong>
+                  {`'${location.pathname}'`.split('').map((letter, i) => (
```

Comment:
Tiny tiny nit: a string cannot possibly be undefined. Why don't we get rid of `''` and pretend we are in a language where `/` and `-` are valid identifier parts? Just an opinion 😄

Reply:
Sorry, I want to go along with the happy vibe of this suggestion but I don't understand what change it's asking for 😅. `.split()` isn't allowed by TypeScript.

Reply:
```suggestion
                  {location.pathname.split('').map((letter, i) => (
```

Like this. Sorry I wasn't clear.

Reply:
Actually never mind, I misread. I thought in `'/foo' is not defined`, `'/foo'` is a string literal, but that's how ESLint's error message actually reports an invalid identifier.